docs/pages/2020_Intro_Week/index.rst

```diff
@@ -23,11 +23,75 @@ This is the content of the `AiiDA virtual tutorial <http://www.aiida.net/aiida-v
 Presentations
 -------------
 
-.. Add links to presentations
+For this virtual tutorial, all presentations have been recorded beforehand, so you can watch them at the time that is most convenient for you.
+However, you are required to watch certain presentations before the hands-on sessions of certain days.
+These are tabulated below:
+
++--------------------+--------------------------------------------------------------------+
+| Tuesday July 7th   | `AiiDA Tutorial Overview`_                                         |
```

Comment:
Would it (design-wise) be nice if the dates are in bold?

Reply:
Myes, it looks nice! ^^